src/hooks/useProof.ts

```diff
@@ -1,56 +1,49 @@
-import { useCallback, useState } from 'preact/hooks'
+import { useCallback } from 'preact/hooks'
+import CreatePasswordProofParams from 'models/CreatePasswordProofParams'
 import CreateProofParams from 'models/CreateProofParams'
 import Messages from 'models/Messages'
 import Signature from 'models/Signature'
 import createAttestationProof from 'helpers/createAttestationProof'
 import createPasswordProof from 'helpers/createPasswordProof'
-import findVerificationId from 'helpers/findVerificationId'
-import isValidProofMessage from 'helpers/isValidProofMessage'
 import postWebViewMessage from 'helpers/postWebViewMessage'
-import requestSignature from 'helpers/requestSignature'
 
 export default function useProof() {
```

Comment:
Why do we need hook, why can't we have 2 separate functions? Your functions don't depend on any params passed to hook/states in hook, so there's no need in hook at all

Reply:
Agree, we can separate functions and use them without any hook